From d7c8daacad4fde2b1997109e0ea5c985840149d2 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:48:44 +0900
Subject: [PATCH 11/23] Revert "Revert "arm64: dts: r8a7795: Add MFIS device
 node""

is reverts commit 8e7c5c5c7fd2 ("Revert "arm64: dts: r8a7795: Add
MFIS device node"")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 3027c49..ffbfab5 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -872,6 +872,14 @@
 			       <&ipmmu_mp0 30>, <&ipmmu_mp0 31>;
 		};
 
+		mfis: mfis@e6260000 {
+			compatible = "renesas,mfis-r8a7795", "renesas,mfis";
+			reg = <0 0xe6260000 0 0x0200>;
+			interrupts = <GIC_SPI 180 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "eicr0";
+			status = "okay";
+		};
+
 		pfc: pfc@e6060000 {
 			compatible = "renesas,pfc-r8a7795";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

